server/models/comments.js

```diff
@@ -32,7 +32,23 @@ const getCommentModel = (sequelize, DataTypes) => {
     });
 
     Comment.belongsTo(db.Articles, {
-      foreignKey: "articleId"
+      foreignKey: "articleId",
+      onDelete: "CASCADE"
+    });
+    Comment.hasMany(db.Reactions, {
+      scopes: {
+        commentLikes: {
+          include: [
+            {
+              model: db.Reactions,
+              where: { sourceType: 2 }
```

Comment:
Please assign this to a constant it is not good practice to hard code values that are non-descriptive. If someone new is looking at this code. He will be wondering what the number 2 is doing there